build/config.gypi

```diff
@@ -0,0 +1,67 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
In a word, no. It's a file that helped me get past the problem I was having when I couldn't run the app on my computer. I added it to my gitignore but only after my first push, so even if I push again it still shows up. Isn't there a way I can remove it in the PR?